homeassistant/components/qbittorrent/sensor.py

```diff
@@ -92,30 +89,77 @@ class QBittorrentSensor(SensorEntity):
 
     def __init__(
         self,
-        description: SensorEntityDescription,
         qbittorrent_client,
         client_name,
         exception,
+        server_unique_id,
+        description: SensorEntityDescription,
     ):
         """Initialize the qBittorrent sensor."""
         self.entity_description = description
         self.client = qbittorrent_client
         self._exception = exception
-
+        self._server_unique_id = server_unique_id
         self._attr_name = f"{client_name} {description.name}"
         self._attr_available = False
 
-    def update(self):
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return f"{self._attr_name}"
+
+    @property
+    def unique_id(self):
+        """Return the unique id of the sensor."""
+        return f"{self._server_unique_id}/{self._attr_name}"
```

Comment:
Name should not be part of a unique ID

See: <https://developers.home-assistant.io/docs/entity_registry_index/#unique-id-requirements>